apps/dg/controllers/app_controller.js

```diff
@@ -588,7 +588,7 @@ DG.appController = SC.Object.create((function () // closure
       if (mimeSpec) {
         switch (mimeSpec.group) {
           case 'TEXT':
-            this.importTextFromUrl(iURL);
+            this.importTextFromUrl(iURL, false, iName);
```

Comment:
If I understand this correctly, then _any_ URL dropped into CODAP will suppress the case table open. Is this right?  Outside the Sage context, its desirable for the case table to be open if a URL is dropped, just as when a CSV file is dropped.

Reply:
The previous code was not passing a value for the second parameter.  I could set it to undefined to exactly match the previous code if you want but I thought false would be better since it is a boolean parameter and both false and undefined are "falsy".